src/main/java/Task.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Task class contains information about the task.
+ */
+public class Task {
+    protected Boolean isDone;
+    protected String icon; //tick or cross
+    protected String name;
+    protected String taskType;
+
+    private final static String TICK = "O";
```

Comment:
Perhaps using the UTF representation of a tick is better than a "O"? Since the variable name is TICK